src/seals/testing/envs.py

```diff
@@ -147,11 +153,11 @@ def test_seed(env: gym.Env, env_name: str, deterministic_envs: Iterable[str]) ->
         AssertionError if test fails.
     """
     env.action_space.seed(0)
-    actions = [env.action_space.sample() for _ in range(10)]
-
+    actions = [env.action_space.sample() for _ in range(rollout_len)]
     # With the same seed, should always get the same result
     seeds = env.seed(42)
-    assert isinstance(seeds, list)
+    assert isinstance(seeds, (list, tuple))
+    # output of env.seed should be a list, but atari environments return a tuple.
```

Comment:
Move this comment one line higher? (comments should usually go before or on the same line as the thing they're explaining.)